src/components/Cart/CartProducts.vue

```diff
@@ -0,0 +1,190 @@
+      <div class="np-cartProducts__item__details__actions">
+        <span :class="{'text-secondary-light': item.quantity === 1}"
+              class="np-cartProducts__item__details__action"
+              @click="decrement(item)"> - </span>
+
+        <p class=" np-cartProducts__item__details__quantity text-secondary-dark"> {{ item.quantity }}</p>
+        <span class="np-cartProducts__item__details__action"
+              @click="increment(item)"> + </span>
+      </div>
+
+      <IconCross class="np-cartProducts__item__details__cross"
+                 @click="showModal = true; product = item"/>
+
+    </div>
+  </div>
+  <div class="np-cartProducts__price">
+    <p>{{ t('totalPrice') }}</p>
+    <p>${{ price.toFixed(2) }}</p>
+  </div>
+
+  <Buttons/>
+
+  <teleport to="#modal">
+    <div v-if="showModal" class="shadow">
+      <div class="np-modal">
+        <p class="np-modal__text">{{ t('deleteProduct') }}</p>
+        <div class="np-modal__wrapper">
+          <button class="np-modal__button"
+                  @click="removeFromCart(product) ; showModal = false">
+            {{ t('yes') }}
+          </button>
+
+          <button class="np-modal__button"
+                  @click="showModal = false">
+            {{ t('no') }}
+          </button>
+        </div>
+
+      </div>
+    </div>
+  </teleport>
+</template>
+
+<script lang="ts">
+import {useStore} from "@/store";
+import {defineComponent, computed, ref, ComputedRef, Ref} from "vue";
+import Buttons from "@/components/Cart/Summary/Buttons.vue";
+import IconCross from "@/assets/icons/icon-cross.vue";
+import {CartItem} from "@/store/interfaces";
+import {useI18n} from "vue-i18n";
+
+export default defineComponent({
+  components: {
+    Buttons,
+    IconCross,
+  },
+  setup() {
+    const store = useStore();
+    const {t} = useI18n();
+    const cart: ComputedRef<CartItem> = computed(() => store.getters['cart/cartItems']);
+    const price: ComputedRef<number> = computed(() => store.getters['cart/priceTotal']);
+
+    const showModal: Ref<boolean> = ref(false);
+    const product: Ref<object> = ref({});
+
+    const removeFromCart = (item: CartItem): void => store.commit('cart/removeFromCart', item)
+    return {
+      t,
+      cart,
+      showModal,
+      product,
+      price,
+      turnHoverOn: (item: CartItem) => store.commit('cart/turnHoverOn', item),
+      turnHoverOff: (item: CartItem) => store.commit('cart/turnHoverOff', item),
+      increment: (item: CartItem) => store.commit('cart/incrementQuantity', item),
+      decrement: (item: CartItem) => store.commit('cart/decrementQuantity', item),
+      removeFromCart,
+    }
+  }
+});
+</script>
+
+<style scoped lang="scss">
+.np-cartProducts__item {
+  @apply w-full flex px-12 py-2;
+
+  &:hover {
+    .np-cartProducts__item__details {
+      background-color: #fff;
+    }
+  }
+
+  &__image {
+    @apply w-16 h-14;
+  }
+
+  &__details {
+    @apply flex items-center justify-between w-full;
+
+
```

Comment:
useles empty spce. have you run lint on this code? should fix this automatically